cypress/e2e/dashboard.cy.js

```diff
@@ -0,0 +1,88 @@
+describe('Dashboard', () => {
+  beforeEach(() => {
+    cy.intercept('GET', 'https://botw-compendium.herokuapp.com/api/v2/category/materials', {fixture: 'items.json'})
+    cy.visit('http://localhost:3000/')
+  });
+  it('should have access to the Zeldas Compendium website', () => {
+  })
+  it('should display the websites logo and name on page load', () => {
+    cy.get('.website-logo')
+      .should('be.visible')
+    cy.get('.website-name')
```

Comment:
Instead of should('be.visible') 
You could do .get('.website-name').contains('Zeldas Compendium')